content/en/docs/setup/production-environment/tools/kubespray.md

```diff
@@ -6,21 +6,27 @@ weight: 30
 
 <!-- overview -->
 
-This quickstart helps to install a Kubernetes cluster hosted on GCE, Azure, OpenStack, AWS, vSphere, Equinix Metal (formerly Packet), Oracle Cloud Infrastructure (Experimental) or Baremetal with [Kubespray](https://github.com/kubernetes-sigs/kubespray).
-
-Kubespray is a composition of [Ansible](https://docs.ansible.com/) playbooks, [inventory](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/ansible.md), provisioning tools, and domain knowledge for generic OS/Kubernetes clusters configuration management tasks. Kubespray provides:
-
-* a highly available cluster
-* composable attributes
-* support for most popular Linux distributions
-  * Ubuntu 16.04, 18.04, 20.04, 22.04
-  * CentOS/RHEL/Oracle Linux 7, 8
-  * Debian Buster, Jessie, Stretch, Wheezy
-  * Fedora 34, 35
-  * Fedora CoreOS
-  * openSUSE Leap 15
-  * Flatcar Container Linux by Kinvolk
-* continuous integration tests
+This quickstart helps to install a Kubernetes cluster hosted on **AWS, GCE, Azure, OpenStack, vSphere, Equinix Metal(bare metal), Oracle Cloud Infrastructure (Experimental), or Baremetal** with [Kubespray](https://github.com/kubernetes-sigs/kubespray).
+
+Kubespray is a composition of [Ansible](https://docs.ansible.com/) playbooks, [inventory](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/ansible.md#inventory), provisioning tools, and domain knowledge for generic OS/Kubernetes clusters configuration management tasks. 
+
+Kubespray provides:
+
+* **Highly available** cluster
+* **Composable** (Choice of the network plugin for instance)
+* Supports most popular **Linux distributions**
+  - **Flatcar Container Linux by Kinvolk**
+  - **Debian** Bullseye, Buster, Jessie, Stretch
+  - **Ubuntu** 16.04, 18.04, 20.04, 22.04
+  - **CentOS/RHEL** 7, 8
+  - **Fedora** 34, 35
+  - **Fedora CoreOS**
+  - **openSUSE** Leap 15.x/Tumbleweed
+  - **Oracle Linux** 7, 8
+  - **Alma Linux** 8
+  - **Rocky Linux** 8
+  - **Amazon Linux 2** 
+* **Continuous integration tests**
```

Comment:
Why are we adding bold format to these texts?

Reply:
With reference to the original kubernetes-sigs/kubespray annotation, it has been adjusted
 [ref 4: Requirements](https://github.com/kubernetes-sigs/kubespray/blob/master/README.md#requirements)